internal/cli/atlas/integrations/describe.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integrations
+
+import (
+	"strings"
+
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type DescribeOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	integrationType string
+	store           store.IntegrationDescriber
+}
+
+func (opts *DescribeOpts) initStore() error {
+	var err error
+	opts.store, err = store.New(config.Default())
+	return err
+}
+
+var describeTemplateSlack = `TYPE	API TOKEN	TEAM	CHANNEL
+{{.Type}}	{{.APIToken}}	{{.TeamName}}	{{.ChannelName}}
+`
+var describeTemplateDatadogOpsGenie = `TYPE	API KEY	REGION
+{{.Type}}	{{.APIKey}}	{{.Region}}
+`
+var describeTemplateFlowdog = `TYPE	API TOKEN	FLOW NAME	ORGANIZATION
+{{.Type}}	{{.APIToken}}	{{.FlowName}}	{{.OrgName}}
+`
+var describeTemplateNewRelic = `TYPE	ACCOUNT ID	LICENSE KEY	WRITE TOKEN	READ TOKEN
+{{.Type}}	{{.AccountID}}	{{.LicenseKey}}	{{.WriteToken}}	{{.ReadToken}}
+`
+var describeTemplatePagerDuty = `TYPE	SERVICE KEY
+{{.Type}}	{{.ServiceKey}}
+`
+var describeTemplateVictorOps = `TYPE	API KEY	ROUTING KEY
+{{.Type}}	{{.APIKey}}	{{.RoutingKey}}
+`
+var describeTemplateWebhook = `TYPE	URL	SECRET
+{{.Type}}	{{.URL}}	{{.Secret}}
+`
+
+func (opts *DescribeOpts) Run() error {
+	r, err := opts.store.Integration(opts.ConfigProjectID(), opts.integrationType)
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(r)
+}
+
+func (opts *DescribeOpts) newDescribeTemplate() string {
```

Comment:
[nit] new goes with a pointer, when I say for go `get`s are not idiomatic I mean just drop the get part and call it by what it is, in this case `describeTemplate` or maybe just `template` 
```suggestion
func (opts *DescribeOpts) newDescribeTemplate() string {
```